neutron/assetlist.json

```diff
@@ -500,6 +500,32 @@
       "images": [
         {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/jimmy.png"
+        },
+        {
```

Comment:
Schema is broken here. This should not be a continuation of Jimmy's images, but rather a new asset.